File: Java/Multithreading.java
import java.io.*;
import java.lang.*;
import java.util.*;

class First extends Thread {             //Thread is a predefined class
    public void run() {       
        int i;
        for(i=1; i<=5; i++) {
            System.out.println("First Thread: "+i);
            try {
                sleep(3000);            //there will be a gap of 3 secs during execution of loop
            }catch(Exception e) {
                System.out.println(e);
            }
        }
    }
}
class Second extends Thread {
    public void run() {
        int i;
        for(i=1; i<=5; i++) {
            System.out.println("Second Thread: "+i);
            try {
                sleep(3000);
            }catch(Exception e) {
                System.out.println(e);
            }
        }
    }
}
class Multithreading {
    public static void main(String[] args) {
        First t1 = new First();
        Second t2 = new Second();
        t1.start();          //start is a predefined function
        t2.start();
    }
}

File: Python/Multithreading.py
# In multithreading thread is a light-weight process. It is beneficial for acheiving multitasking.

# CPU has multi-core processor. It helps to do multiple tasks in different cores.

from time import sleep
from threading import *
class FirstProgram(Thread):
    def run(self):
        for i in range(6):
            print("Hello!")
            sleep(3)        #the thread will run with 3 sec difference

class SecondProgram(Thread):
    def run(self):
        for i in range(6):
            print("How are you?")
            sleep(3)

obj1 = FirstProgram()       #objects are created
obj2 = SecondProgram()

obj1.start()      #this will start the thread
obj2.start()

obj1.join()
obj2.join()

print("Thankyou!")